nym-vpn-core/nym-vpn-lib/uniffi/nym_vpn_lib.kt

```diff
@@ -872,7 +872,7 @@ private fun uniffiCheckContractApiVersion(lib: UniffiLib) {
 
 @Suppress("UNUSED_PARAMETER")
 private fun uniffiCheckApiChecksums(lib: UniffiLib) {
-    if (lib.uniffi_nym_vpn_lib_checksum_func_checkcredential() != 37960.toShort()) {
+    if (lib.uniffi_nym_vpn_lib_checksum_func_checkcredential() != 2527.toShort()) {
```

Comment:
Where does the magic number come from?

Reply:
Uniffi generated. I have no idea where it comes from.